corehq/apps/registration/templates/registration/domain_request.html

```diff
@@ -1,8 +1,23 @@
 {% extends "registration/base.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
 
 {% block title %}New Project Space{% endblock title %}
 
+{% block js %}{{ block.super }}
+    <script src="{% static 'hqwebapp/js/lib/select2/select2.js' %}"></script>
+{% endblock %}
+
+{% block head %}{{ block.super }}
+    <link href="{% static 'hqwebapp/js/lib/select2/select2.css' %}" rel="stylesheet" />
+{% endblock %}
```

Comment:
Haha, was thinking, may be this snippet is used too often and we should make a custom tag or something like that. Will add that.